addons/tagging/functions/fnc_tag.sqf

```diff
@@ -20,7 +21,8 @@
 params [
     ["_unit", objNull, [objNull]],
     ["_texture", "", [""]],
-    ["_material", "", [""]]
+    ["_material", "", [""]],
+    ["_tagModel", "UserTexture1m_F", [""]]
```

Comment:
This is not used anywhere in this function. Should be added to L117 so it is passed to `createTag`.

Reply:
Its not used in the params, but should still be in the header, and adding it to params will make sure that it prints error at correct place if you pass wrong datatype